Automattic/jetpack
# Update/add better free check for my jetpack products

## Proposed changes:

* This diff adds a boolean value to My Jetpack products to indicate if they have a free offering

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* There are no functional tests - just make sure My Jetpack still loads with this change at that the code looks correct for which products have a free offering
